docs/observability/aws/integrations/aws-ec2-host-metrics.md

```diff
@@ -58,7 +59,7 @@ Use this dashboard to:
 * Identify trends and deviations in resource usage across instance types, based upon which you can identify which instance types need to be resized.
 * Monitor average CPU utilization by instance type.
 
-![1.2 AWS EC2 Overview (Host OS metrics).png](https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/Host-Metrics-EC2/1.2.-AWS-EC2-Overview-Host-OS-metrics.png)
+<img src={useBaseUrl('img/observability/1.2.-AWS-EC2-Overview-Host-OS-metrics.png')} alt="AWS_EC2_Overview"/>
 
 ### 1.2 AWS EC2 Summary (Host OS metrics)
```

Comment:
If matching UI, complete match, otherwise use generic headings without numbers. Please check none of the headings match the dashboard names.

Reply:
The numbers are added as per the request and these numbers will be inline with the dashboard title. Also, the H4 dashboard titles looks to be inline with the title in newly added dashboard image.

Reply:
H4 - 1.2 AWS EC2 Summary (Host OS metrics). This shows you are trying to match dashboard names, but this is incorrect. In the dashboard, there is a dot (.) after 2.

Reply:
comments fixed.